tests/test_discretisation/test_decision_tree_discretiser.py

```diff
@@ -106,6 +106,24 @@ def test_non_fitted_error(df_vartypes):
         transformer.transform(df_vartypes)
 
 
+@pytest.fixture(scope="module")
+def df_discretise():
+    np.random.seed(42)
+    mu1, sigma1 = 0, 3
+    s1 = np.random.normal(mu1, sigma1, 20)
+    mu2, sigma2 = 3, 5
+    s2 = np.random.normal(mu2, sigma2, 20)
+    data = {
+        "var_A": s1,
+        "var_B": s2,
+        "target": [0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 1, 1, 1, 1]
+    }
+
+    df = pd.DataFrame(data)
+
+    return df
+
+
```

Comment:
Why do is `df_discterise()` in the test file and not the conftest file?

Reply:
in the general conftest we put dataframes that are used for testing across modules, by more than 1 transformer.

This df you created was exclusive to test this discretizer. So it is better to put it in the test file it is used.